package.json

```diff
@@ -40,7 +40,7 @@
     "test:render": "npx patch-package && jest test/render.test.ts --forceExit",
     "test:unit": "jest --testPathPattern src",
     "test:xmlserver": "eleventy --serve --port=8085",
-    "test:validate-xml": "eleventy; find _examples_site test/schema -name '*.xml' -not -path '*/advanced_behaviors/instawork.xml' -not -path '*/advanced_behaviors/custom_share.xml' -not -path '*/advanced_behaviors/alert/behavior.xml' -not -path '*/case_studies/business_rating.xml' -not -path '*/case_studies/_rating_save.xml' -not -path '*/index.xml' -not -path '*/index_hyperview.xml' | xargs xmlschema-validate --schema schema/hyperview.xsd --version 1.1",
+    "test:validate-xml": "eleventy; find demo/hyperview/public test/schema -name '*.xml' -not -path '*/advanced/scroll/*.xml' -not -path '*/advanced_behaviors/instawork.xml' -not -path '*/advanced_behaviors/custom_share.xml' -not -path '*/advanced_behaviors/alert/behavior.xml' -not -path '*/case_studies/business_rating.xml' -not -path '*/case-studies/business-rating/index.xml' -not -path '*/case-studies/business-rating/rating_save.xml'  -not -path '*/case_studies/_rating_save.xml' -not -path '*/case-studies/photos/comments.xml' -not -path '*/index.xml' -not -path '*/index_hyperview.xml' | xargs xmlschema-validate --schema schema/hyperview.xsd --version 1.1",
```

Comment:
we should clean up the list of `-not -path` to remove files that no longer exist (because their location has moved)

Reply:
addressing this in another PR: https://github.com/Instawork/hyperview/pull/927